docker-compose.yml

```diff
@@ -143,7 +143,7 @@ services:
         volumes:
             - ./tests/snapshots:/snapshots
         environment:
-            - LOG_LEVEL=WARNING
+            - LOG_LEVEL=DEBUG
```

Comment:
was this change done for testing? Should it be moved back to WARNING?

Reply:
Yeah, was just changed for testing. [Here](https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/34647/workflows/874f1af1-737c-4f57-bdd2-6e59dcf538b1/jobs/2342911?invite=true#step-107-1306) is an example of the logs when set to debug, they are now available for all integration tests on circleci.

Reply:
Will switch this back before the final commit